sound/soc/sof/loader.c

```diff
@@ -305,6 +305,9 @@ static int snd_sof_fw_ext_man_parse(struct snd_sof_dev *sdev,
 		case SOF_EXT_MAN_ELEM_CONFIG_DATA:
 			ret = ext_man_get_config_data(sdev, elem_hdr);
 			break;
+		case SOF_EXT_MAN_ELEM_PLATFORM_CONFIG_DATA:
+			/* this is platform specific data, no need to print warning */
```

Comment:
why not call your `.parse()` hook here? Wouldn't it be easier instead of scanning the whole manifest yourself?

Reply:
thanks! I missed this point